apps/site/src/components/pages/dashboard/utils.ts

```diff
@@ -122,7 +122,7 @@ export const getDashboardSectionCards = (props: {
           "View and update the types you’ve created and made public on the Hub",
         link: {
           title: "Manage types",
-          href: `${props.profileLink}/schemas`,
+          href: `${props.profileLink}/all-types`,
```

Comment:
@CiaranMn Why `all-types` and not just `types`?

Reply:
There was a clash with the `types/entity-type` etc routes – I can have a look at sorting it out later in the week

Reply:
https://app.asana.com/0/1203358502199087/1204015967491453/f